dynawo/sources/Models/Modelica/Dynawo/Examples/ENTSOE/TestCase1.mo

```diff
@@ -0,0 +1,138 @@
+  Dynawo.Electrical.Controls.Machines.PowerSystemStabilizers.IEEE_PSS2A pss(IC1 = 1, IC2 = 3, Ks1 = 10, Ks2 = 0.1564, Ks3 = 1, PGen0Pu = -generatorSynchronous.P0Pu, PNomAlt = generatorSynchronous.PNomAlt, T1 = 0.25, T2 = 0.03, T3 = 0.15, T4 = 0.015, T6 = 1e-5, T7 = 2, T8 = 0.5, T9 = 0.1, Tw1 = 2, Tw2 = 2, Tw3 = 2, Tw4 = 1e-5, Upss0Pu = 0, VstMax = 0.1, VstMin = -0.1) annotation(
+    Placement(visible = true, transformation(origin = {90, 0}, extent = {{-10, 10}, {10, -10}}, rotation = 0)));
+  Dynawo.Electrical.Controls.Machines.Governors.IEEE_TGOV1 governor(Dt = 0, Pm0Pu = generatorSynchronous.Pm0Pu, R = 0.05, Tg1 = 0.5, Tg2 = 3, Tg3 = 10, VMax = 1, VMin = 0) annotation(
+    Placement(visible = true, transformation(origin = {90, -30}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Sources.Step step(height = 0.05, offset = avr.UsRef0Pu, startTime = 0.1) annotation(
+    Placement(visible = true, transformation(origin = {10, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+equation
+  connect(generatorSynchronous.omegaRefPu, Omega0Pu.setPoint) annotation(
+    Line);
+  generatorSynchronous.switchOffSignal1.value = false;
+  generatorSynchronous.switchOffSignal2.value = false;
+  generatorSynchronous.switchOffSignal3.value = false;
+  connect(currentBus.terminal, generatorSynchronous.terminal) annotation(
+    Line(points = {{-120, 0}, {20, 0}}, color = {0, 0, 255}));
+  connect(generatorSynchronous.omegaPu_out, governor.omegaPu) annotation(
+    Line(points = {{38, -6}, {60, -6}, {60, -36}, {78, -36}}, color = {0, 0, 127}));
+  connect(generatorSynchronous.PGenPu_out, pss.PGenPu) annotation(
+    Line(points = {{38, 10}, {70, 10}, {70, 6}, {78, 6}}, color = {0, 0, 127}));
+  connect(generatorSynchronous.omegaPu_out, pss.omegaPu) annotation(
+    Line(points = {{38, -6}, {78, -6}}, color = {0, 0, 127}));
+  connect(pss.UpssPu, avr.UpssPu) annotation(
+    Line(points = {{101, 0}, {110, 0}, {110, 12}, {118, 12}}, color = {0, 0, 127}));
+  connect(generatorSynchronous.UsPu_out, avr.UsPu) annotation(
+    Line(points = {{38, 18}, {118, 18}}, color = {0, 0, 127}));
+  connect(step.y, avr.UsRefPu) annotation(
+    Line(points = {{21, 60}, {70, 60}, {70, 24}, {118, 24}}, color = {0, 0, 127}));
+  connect(governor.PmPu, generatorSynchronous.PmPu_in) annotation(
+    Line(points = {{102, -30}, {102, -30.5}, {110, -30.5}, {110, -51}, {32, -51}, {32, -16}}, color = {0, 0, 127}));
+  connect(avr.EfdPu, generatorSynchronous.efdPu_in) annotation(
+    Line(points = {{141, 18}, {150, 18}, {150, -60}, {8, -60}, {8, -16}}, color = {0, 0, 127}));
+  annotation(preferredView = "diagram",
+    experiment(StartTime = 0, StopTime = 2, Tolerance = 1e-06),
+    __OpenModelica_simulationFlags(initialStepSize = "0.001", lv = "LOG_STATS", nls = "kinsol", s = "ida", nlsLS = "klu", maxIntegrationOrder = "2", maxStepSize = "10", emit_protected = "()"),
+    Diagram(coordinateSystem(extent = {{-160, -100}, {160, 100}})));
```

Comment:
It would have been very nice to include in the navigation view the plots obtained for the test case that are used in the ENTSO-E report.
It would have been also very nice to refer to the document (url) in this part so that the reference is very clear.